core/src/docs/rfcs/3574_concurrent_stat_in_list.md

```diff
@@ -0,0 +1,62 @@
+- Proposal Name: `concurrent_stat_in_list`
+- Start Date: 2023-09-16
+- RFC PR: [apache/incubator-opendal#3574](https://github.com/apache/incubator-opendal/pull/3574)
+- Tracking Issue: [apache/incubator-opendal#3097](https://github.com/apache/incubator-opendal/issues/3097)
```

Comment:
We need to open a new issue to monitor the implementation of this RFC.

Reply:
Done